components/telemetry-operator/api/v1alpha1/logpipeline_types.go

```diff
@@ -47,12 +46,12 @@ type MultiLineParser struct {
 
 // Filter describes a Fluent Bit filter configuration section
```

Comment:
Maybe we remove the "section" from the doc string. Having a raw config section in here will be the exception in future. Let's make it "Filter describes a Fluent Bit filter configuration". Same for output.